src/CharucoBoardDetector.cpp

```diff
@@ -68,6 +74,21 @@ void CharucoBoardDetector::onImageImpl(const sensor_msgs::ImageConstPtr &img) {
     cameraToBoardStamped.transform = tf2::toMsg(cameraToBoard);
 
     broadcaster->sendTransform(cameraToBoardStamped);
+
+    geometry_msgs::Vector3Stamped positionMsg;
```

Comment:
What is the use case for the `position` publisher? I think the user can listen to the `pose` and just take the translation part